panther_controller/config/WH05_controller.yaml

```diff
@@ -0,0 +1,84 @@
+/**:
```

Comment:
If there are plans to create a lynx controller, you can create it now. If you can resolve the comment.

Reply:
The `WH05_controller.yaml` is intended to be a default lynx controller. WH05 most likely will be name of default lynx wheels. For now I placed it in panther_controller package, but later a dedicated package for lynx will be created.